ios/PacketTunnel/TunnelMonitor/TunnelMonitor.swift

```diff
@@ -312,7 +312,7 @@ final class TunnelMonitor: PingerDelegate {
         }
     }
 
-    func onSleep(completion: @escaping () -> Void) {
```

Comment:
Sorry for my lack of knowledge in Swift, but does something similar to this lint rule exist?
It would help catching this kind of unused symbols.

https://eslint.org/docs/latest/rules/no-unused-vars

Reply:
Thanks for sharing @buggmagnet .